pkg/restic/backupper.go

```diff
@@ -162,6 +170,12 @@ func (b *backupper) BackupPodVolumes(backup *velerov1api.Backup, pod *corev1api.
 			continue
 		}
 
+		// volumes that are not mounted by any container should not be backed up, because
+		// its directory is not created
+		if !mountedPodVolumes.Has(volumeName) {
+			continue
```

Comment:
add a log like `volume xxx is not mounted to pod xxx/xxx, skipping` ?

Reply:
Yeah, that'll be useful, I'll add it.